### DISCUSSION TITLE
Conversion from 2 to 3

### DISCUSSION THREAD

**Discussion by moseley1983** [nornir-automation/nornir#703] [Q&A]
Hi all, I'm trying to convert the below line from Nornir 2 to Nornir 3 but can't find the 3 equivalent:

from nornir.plugins.tasks import networking, text

The v3.1.1 documentation is not ready yet at https://nornir.readthedocs.io/en/v2.5.0/plugins/tasks/networking.html

Any ideas please?

**Comment by ktbyers** [comment]
This should be the Nornir 3.x documentation

https://nornir.readthedocs.io/en/latest/

The specific page you are referring to no longer exists in the Nornir 3.x documentation. 

The particular issue you are running into is because Nornir3 no longer contains the plugins (nornir-core no longer contains the plugins). These are in separately installable packages.

See the first item here:

https://github.com/twin-bridges/nornir_course/blob/master/nornir3_changes.md

The plugin directory is here:

https://nornir.tech/nornir/plugins/
